### DISCUSSION TITLE
Activation of the student developer pack

### DISCUSSION THREAD

**Discussion by dhanush1092003** [community/community#65271] [GitHub Education]
**my application is approved, how much time it takes to pack get activated**

**Comment by nirav-gajera** [comment]
It usually takes 1-2 business days for the Student Developer Pack to be activated after your application is approved. However, it can take longer during peak times. after activating the PRO badge will appears in your profile

**Reply by dhanush1092003** [reply]
Is it sure becoz we should do projects by sep 5

**Reply by nirav-gajera** [reply]
My developer pack was approved within 30 minutes. However, the approval time depends on your submit application and the traffic during the new semester.

**Comment by wrdspruwir72** [comment]
I have the same question

**Reply by jesse-sealand** [reply]
@wrdspruwir72, this question has been answered. Do you need more information beyond @nirav-gajera's answer?

**Comment by Binary-Ninja-007** [comment]
Good.

**Comment by SupratickDey** [comment]
My Application was approved but still student Developer Pack  is not activated
